src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { RadioGroup } from "./RadioGroup";
+import { TextArea } from "../TextArea/TextArea";
+
+export default {
+  title: "USWDS/Base/RadioGroup",
+  component: RadioGroup,
+  argTypes: {
+    groupName: {
+      description:
+        "Name of the radio group. This name automatically propagates to each Radio component in a RadioGroup.",
+    },
+    radioProps: {
+      description:
+        "An array of Radio component properties. See the Radio component for a complete list.",
+    },
+  },
```

Comment:
Do you think there's value in keeping the radioProps data exposed here? 

If you think there's no value, then we could hide it with

`control: false,`

Reply:
seconded